triggers/service/auth/src/test/scala/com/daml/auth/middleware/oauth2/TestMiddleware.scala

```diff
@@ -540,10 +543,12 @@ class TestMiddlewareClientNoRedirectToLogin
     with TryValues
     with TestFixture
     with SuiteResourceManagementAroundAll {
+  val authenticationSecurity: SecurityTest =
+    SecurityTest(property = Authentication, asset = "TBD")
+
   override protected val redirectToLogin: Client.RedirectToLogin = Client.RedirectToLogin.No
   "the authorize client" should {
```

Comment:
`the TestMiddlewareClientNoRedirectToLogin client`

It's important to point out the client used, because otherwise we get:
- Test case 1: should not redirect
- Test case 2: should redirect
which sounds contradictory to the uninitiated observer.